core/src/main/java/org/lflang/generator/docker/DockerGenerator.java

```diff
@@ -150,6 +152,27 @@ protected String generateCopyOfScript() {
     return "# (No pre-run script provided.)";
   }
 
+  /**
+   * Return zero or more COPY commands to copy files specified using the {@code files} target
+   * property from the builder to the runner.
+   */
+  protected String generateCopyOfUserFiles() {
+    var files = context.getTargetConfig().get(FilesProperty.INSTANCE);
```

Comment:
As evidenced by test failure, this won't work for all targets. Put it behind a check `context.getTargetConfig.isSupported(FilesProperty.INSTANCE)`